sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/init.sql

```diff
@@ -1,3 +1,25 @@
+-- contains_qualified_fxa_activity_event
+-- Expects an array of event_types and service name
+-- Iterates and checks if an event_type, service combination
+-- should be considered when determining if this is the first
+-- time it has been observed.
+-- Example usage:
+-- contains_qualified_fxa_activity_event(['fxa_activity'], NULL),  # returns TRUE
+-- contains_qualified_fxa_activity_event(['fxa_activity'], 'firefox-ios'),  # returns TRUE
+-- contains_qualified_fxa_activity_event(['fxa_login - complete'], NULL),  # returns FALSE
+-- contains_qualified_fxa_activity_event(['fxa_login - complete'], 'firefox-ios'),  # returns TRUE
+CREATE TEMP FUNCTION contains_qualified_fxa_activity_event(events ANY TYPE, `service` ANY TYPE) AS (
```

Comment:
rather than duplicate all of the logic from query.sql to init this table, it's probably worth making this query something that would only init the schema, such as a `SELECT ... WHERE FALSE` type query, like here https://github.com/mozilla/bigquery-etl/blob/d01ce2fc8a6193121dc961951c850cf4bf8bcc3e/sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v1/init.sql#L3-L18

Reply:
If we take the approach to init the schema only then `init.sql` is probably redundant assuming schema.yaml is accurately reflecting the table structure?

Reply:
if i understand correctly, having an `init.sql` that produces an accurate schema allows the tooling to automatically update the `schema.yaml` when new fields are added to recursive tables (like clients_last_seen). not sure if that's relevant here, if it's not recursive probably leave out init.sql entirely.

Reply:
Thanks, I just decided to delete it completely (at least for now).